src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Windows.targets

```diff
@@ -167,4 +167,72 @@ Copyright (c) .NET Foundation. All rights reserved.
                       and '$(UseUwp)' == 'true' ">
     <NetSdkError ResourceName="WindowsSDKXamlInvalidTfm" />
   </Target>
+
+  <!--
+    ============================================================
+                        _RunCsWinRTGenerator
+
+    Runs 'cswinrtgen' to produce the 'WinRT.Interop.dll' assembly.
+    ============================================================
+  -->
+  <UsingTask Condition="'$(CsWinRTGeneratorTasksOverriden)' != 'true'" TaskName="RunCsWinRTGenerator" AssemblyFile="$(MicrosoftNETBuildTasksAssembly)" />
+  <Target
+    Name="_RunCsWinRTGenerator"
+    DependsOnTargets="CoreCompile;$(GetTargetPathDependsOn);$(GetTargetPathWithTargetPlatformMonikerDependsOn)"
+    BeforeTargets="GetTargetPath;GetTargetPathWithTargetPlatformMoniker;GenerateBuildDependencyFile;GeneratePublishDependencyFile"
+    Condition="'$(CsWinRTGenerateInteropAssembly)' == 'true'">
+    
+    <!-- Default property values for 'cswinrtgen' -->
+    <PropertyGroup>
+      <CsWinRTToolsDirectory Condition="'$(CsWinRTToolsDirectory)' == ''"></CsWinRTToolsDirectory>
```

Comment:
Here we need to get the path to the root directory of the `Microsoft.Windows.SDK.NET.Ref` package that's going to be used. Not sure how to get it from here though.

Reply:
If there's a packageref to this package (implicit or explicit), there may be a `PackagePath_MIcrosoft_Windows_SDK_NET_Ref` MSbuild property already ambiently available. Check a binlog and see?

Reply:
Tried in a blank app targeting `net9.0-windows10.0.26100.0`, not seeing that variable 🥲
Is this something we could configure in the .NET SDK? Since it's the one adding this 🤔